src/condor_utils/dag_commands.cpp

```diff
@@ -119,3 +119,44 @@ const char* DAG::GET_SCRIPT_DEBUG_CAPTURE_TYPE(const ScriptOutput type) {
 	auto it = std::ranges::find_if(SCRIPT_DEBUG_MAP, [&type](const auto& pair) { return pair.second == type; });
 	return (it == SCRIPT_DEBUG_MAP.end()) ? "NONE" : it->first.c_str();
 }
+
+std::map<std::string, int> DAG::STRING_SPACE::__string_space_map{};
+
+std::string_view DAG::STRING_SPACE::DEDUP(const std::string_view str) {
+	return DAG::STRING_SPACE::__DEDUP(str, false);
+}
+
+std::string_view DAG::STRING_SPACE::__DEDUP(const std::string_view str, bool internal) {
+	if ( ! str.data()) { return {}; }
+
+	auto it = __string_space_map.find(str.data());
+	if (it != __string_space_map.end()) {
+		if ( ! internal) {
```

Comment:
```suggestion
		if (! internal) {
```
Or
```suggestion
		if ( ! internal ) {
```
but my OCD is being triggered by the space being unbalanced. ;)